101-cosmosdb-create-arm-template/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "apiType": {
+      "value": "Cassandra"
```

Comment:
remove this and use a defaultValue in the template

Reply:
Done